src/Geta.NotFoundHandler.Admin/Areas/GetaNotFoundHandlerAdmin/Pages/Ignored.cshtml.cs

```diff
@@ -40,8 +44,17 @@ public IActionResult OnPostUnignore(string oldUrl)
 
     private void Load()
     {
-        var items = _redirectsService.GetIgnored().ToPagedList(Paging.PageNumber, Paging.PageSize);
+        var items = FindRedirects().ToPagedList(Paging.PageNumber, Paging.PageSize);
         Message = $"There are currently {items.TotalItemCount} ignored suggestions stored.";
         Items = items;
     }
+
+    private List<CustomRedirect> FindRedirects()
+    {
+        var result = _redirectsService.GetIgnored();
+
+        result = Sort(result);
```

Comment:
This could easily be changed into extension, then you could chain the invokation.